MinimOsd_Extra/protocols/NMEA.h

```diff
@@ -150,23 +151,22 @@ bool parse_NMEA_char(char c, char *string) {
         switch(param) { // words in sentence
         case 0:    //frame identification
             frame = 0;
-            if (string[0] == 'G' && string[2] == 'G' && string[3] == 'G' && string[4] == 'A') frame = FRAME_GGA;
-            if (string[0] == 'G' && string[2] == 'R' && string[3] == 'M' && string[4] == 'C') frame = FRAME_RMC;
+            if (string[0] == 'G' && string[1] == 'P' && string[2] == 'G' && string[3] == 'G' && string[4] == 'A') frame = FRAME_GGA;
+            if (string[0] == 'G' && string[1] == 'P' && string[2] == 'R' && string[3] == 'M' && string[4] == 'C') frame = FRAME_RMC;
             break;
 
 /*
- $GPGGA,123519,4807.038,N,01131.000,E,1,08,0.9,545.4,M,46.9,M,,*47
+ $GPGGA,123519,4807.038,N,01131.000,E,1,08,0.9,545.4,M,46.9,M,,*47 //this is wrong string because:
+ - Parameter Time have decimal dot and 2 decimal digits, not only HHMMSS
+ - Parameter Lat have 5 fractional digits, not 3
+ - Parameter Lon have 5 fractional digits, not 3
 
```

Comment:
I did not think you chip is wrong, the string is wrong. Only thing I try is to point to possible problem in parsing this string. Now I am searching for meaning of atof()...